custom-login/README.md

```diff
@@ -30,7 +30,14 @@ mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.client.orgUrl=https://{yourOktaDomain}
```

Comment:
Is this value needed? It doesn't seem to be.

Reply:
@bdemers ^^

Reply:
Good call! That shouldn't be needed any more!